lib/db/schema/patch-083-084.sql

```diff
@@ -0,0 +1,57 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('83');
+
+-- Since we are altering the size of the `recoveryKeyId` column
+-- we need to make sure that the column is empty before it can be applied
+DELETE from recoveryKeys;
+
+ALTER TABLE recoveryKeys MODIFY COLUMN recoveryKeyId BINARY(16);
+
+CREATE PROCEDURE `createRecoveryKey_2` (
+  IN `uidArg` BINARY(16),
+  IN `recoveryKeyIdArg` BINARY(16),
```

Comment:
I decided to alter the size of the column to match the exact size we are sending from https://github.com/mozilla/fxa-content-server/pull/6323. This was previously `BINARY(64)`.